src/index.ts

```diff
@@ -132,8 +132,12 @@ class ServerlessOfflineSns {
         await this.unsubscribeAll();
         this.debug("subscribing");
         await Promise.all(Object.keys(this.serverless.service.functions).map(fnName => {
-            const fn = this.serverless.service.functions[fnName];
-            return Promise.all(fn.events.filter(event => event.sns != null).map(event => {
+            const fn = this.serverless.service.functions[fnName];            
+            return Promise.all(fn.events.filter(event => event.sns != null || event["sqs-from-sns"]).map(event => {
+                if (event["sqs-from-sns"]) {
+                    const sns = event["sqs-from-sns"];
+                    return this.subscribe(fnName, {...sns, arn: sns.topicArn});
```

Comment:
this will go directly from `sns` to `lambda`, correct? No `sqs`

Reply:
yes. but `tiddyMessages` can cope with that